maraudersmap/schemas.py

```diff
@@ -35,5 +54,75 @@ class ItemCreate(ItemBase):
 class Item(ItemBase):
     id: UUID
 
+
+class ItemOwnershipBase(OrmBase):
+    obtained_at: datetime
+
+
+class ItemOwnership(ItemOwnershipBase):
+    id: UUID
+    item: ItemBase
+    owner: UserBase
+
+
+class ItemOwnershipCreate(ItemOwnershipBase):
+    owner_id: UUID = Field(foreign_key="user.id")
+    item_id: UUID = Field(foreign_key="item.id")
+
+
+class QuestItemBase(OrmBase):
+    item: Item
+    location: LatLong
+    unlock_method: UnlockMethod
+
+
+class QuestItemCreate(QuestItemBase):
+    pass
+
+
+class QuestItem(QuestItemBase):
+    id: UUID
+    item: ItemBase
+    quest: QuestBase
+
     class Config:
         orm_mode = True
+
+
+class QuestBase(OrmBase):
+    title: str
+    description: str
+    active_from: datetime
+    active_to: datetime | None = None
+    unlock_method: UnlockMethod
+
+
+class QuestCreate(QuestBase):
+    event_id: UUID
+
+
+class QuestDependencyBase(OrmBase):
+    quest_to_finish_before_id: UUID = Field(foreign_key="quest_to_finish_before_id")
+    quest_to_finish_after_id: UUID = Field(foreign_key="quest_to_finish_after_id")
+
+
+class Quest(QuestBase):
+    id: UUID
```

Comment:
Switch position of these?
```suggestion
class Quest(QuestBase):
    id: UUID


class QuestDependencyBase(OrmBase):
    quest_to_finish_before_id: UUID = Field(foreign_key="quest_to_finish_before_id")
    quest_to_finish_after_id: UUID = Field(foreign_key="quest_to_finish_after_id")
```